client/core/core.go

```diff
@@ -64,6 +65,9 @@ var (
 	// When waiting for a wallet to sync, a SyncStatus check will be performed
 	// ever syncTickerPeriod. var instead of const for testing purposes.
 	syncTickerPeriod = 10 * time.Second
+	// serverAPIVers are the DEX server API versions this client is capable
+	// of communicating with.
+	serverAPIVers = []int{serverdex.InitialAPIVersion}
```

Comment:
This makes sense and I'm OK with using a `server/dex` import for this now, but I suspect we're going to want to break the tight coupling of client -> server packages at some point.

Probably we're just discovering things (like server/account) that would be better in the "common"  `decred.org/dcrdex/dex` packages at some point.